tests/Google/Ads/GoogleAds/InstantiateClassesTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/*
+ * Copyright 2018, Google Inc.
```

Comment:
Need updates to use the latest copyright.

Reply:
Done.